app/lib/features/chat/widgets/create_chat.dart

```diff
@@ -706,18 +706,16 @@ class _UserWidget extends ConsumerWidget {
               style: Theme.of(context).textTheme.labelMedium,
             ),
       leading: avatarLoader.when(
-        data: (avatar) {
```

Comment:
I'd still like to avoid having to review _several_ files with just minor code style changes that are unrelated to the actual PR ... please put those in a separate PR, mark that as `minor` and just merge it, there is no reason for anyone actually reviewing that so its a waste of time...